tests/test_decompiler.py

```diff
@@ -28,6 +30,12 @@
 )  # this variable controls whether we print the decompilation code or not
 
 
+def set_decompiler_option(params: List[str], values):
```

Comment:
To get rid of the `None` check you do everywhere, please re-do this function do be something like `update_decompiler_options(decompiler_options: List[Tuple], params: List[Tuple]): -> List[Tuple]`, where the you take the current decompiler options, check if it is None, if it is return None. If not, take the params list of tuples and update each option.

For your use case, I would hope to see this in every `Decompiler` call in the tests:
```python
...
options=update_decompiler_options(decompiler_options, [("cstyle_ifs", False)])
...
```